packages/assets-controllers/src/assetsUtil.ts

```diff
@@ -432,7 +432,7 @@ export async function fetchTokenContractExchangeRates({
     Hex,
     Awaited<ReturnType<AbstractTokenPricesService['fetchTokenPrices']>>
   >({
-    values: tokenAddresses,
+    values: [...tokenAddresses].sort(),
```

Comment:
I allocated a new array because it's undesirable for a function to modify params passed by the caller.  Caller may not expect it to mutate.  But don't feel strongly if someone prefers in-place sort.